sql/01-create-table-student.sql

```diff
@@ -1 +1,5 @@
-CREATE TABLE students(id INT, name TEXT, grade INT);
+psql sqlexercises; -- To get into database
+
+CREATE SCHEMA student; --To create the shcema for the project. 
```

Comment:
i dont believe creating a schema is necessary.
https://www.postgresql.org/docs/9.3/static/sql-createschema.html